### PR TITLE
implemented The Scenic Hotel Business scenario

### PR DESCRIPTION
**Pull request by Estyg100**

### CODE REVIEW

**Comment by Estyg100** [comment]
I did the work. Pls. review and let me know if I need to make any changes.
Thanks.

File: 1 table.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        constraint ck_Guest_expiration_date_must_be_in_the_format_of_the_month_and_year_separated_by_slash check(ExpirationDate like '[0-1][0-9]/[0-9][0-9]'),
+    RoomType varchar(15) not null 
+        constraint ck_Guest_room_type_must_be_one_of_the_4_possible_options 
+            check(RoomType in('Suite', 'Standard Room', 'Connecting Room', 'Senior Room')),
+    RoomNumber int not null
+        constraint ck_Guest_room_number_must_be_between_100_and_399 check(RoomNumber between 100 and 399),
+    DateCheckedIn date not null
+        constraint ck_Guest_date_checked_in_must_be_after_oppening_date_and_cannot_be_future check(DateCheckedIn between '1998-01-01' and getdate()),
+    DateCheckedOut date null 
+        constraint d_Guest_date_checked_out default null
```
Comment (ChavyWal) [inline 1 table.sql]: I am almost sure that there is no need to specify a default for a null its a default anyway.

File: 3 reports.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+select AvgDaysSpentInHotel = avg(AmountOfDaysSpentInHotel) 
+from guest g
+
+--3) I would like to have a list of those who visited in the summer, stayed for over seven days and were 2 or more people.
+
+select * 
+from guest g 
+where g.Season = 'Summer'
+and AmountOfDaysSpentInHotel > 7 
+and AmountOfPeople > 2
```
Comment (ChavyWal) [inline 3 reports.sql]: I think it was a careless mistake but its supposed to be **two or more people**.

File: 3 reports.sql [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+--4) Show me a list of those Guest who came back more than once.
+
```
Comment (ChavyWal) [inline 3 reports.sql:27]: _This one was a little tricky as only those who came back twice are supposed to be listed not those who took more than one room._

File: 1 table.sql (hunk 1 above)
Comment (Estyg100) [reply 1 table.sql]: I removed it

File: 3 reports.sql (hunk 2 above)
Comment (Estyg100) [reply 3 reports.sql]: I corrected it

File: 3 reports.sql (hunk 3 above)
Comment (Estyg100) [reply 3 reports.sql:27]: I corrected this as well

**Comment by Estyg100** [comment]
I made corrections. Pls. review and let me know.

File: 3 reports.sql (hunk 3 above)
Comment (ChavyWal) [reply 3 reports.sql:27]: I Think your select statement deos not show the right results it is quite complicating but **you can do it:)**

Comment (Estyg100) [reply 3 reports.sql:27]: Guess what? I figured it out! Hope it's correct this time around:)
Pls. let me know.........

Comment (ChavyWal) [reply 3 reports.sql:27]: **Yup you did it the right way this time. amazing!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!**

**Comment by ChavyWal** [comment]
I reviewed your work great job!

**Review by ChavyWal — APPROVED** [review]
great job!

**Pull request closed without merging**